### DISCUSSION TITLE
What are your plans and roadmap for supporting Intel's TDX

### DISCUSSION THREAD

**Discussion by fffmonkeyking** [gramineproject/gramine#1464] [Q&A]
### Description of the feature

What are your plans and roadmap for supporting Intel's TDX ？

### Why Gramine should implement it?

no reason.
it's just your plan.

**Comment by kailun-qin** [comment]
Thanks for your question and interest!

> What are your plans and roadmap for supporting Intel's TDX ？

It's currently not planned in Gramine's roadmap. For detailed info, pls see https://github.com/orgs/gramineproject/projects/1.

Pls kindly note that we've already had a feature request for Gramine to support Intel TDX https://github.com/gramineproject/gramine/issues/1323, you may want to provide more inputs about your motivations/use cases/requirements there.

Since this is basically a question rather than an issue of Gramine, I'll move it to discussions.

**Comment by CasellaJr** [comment]
Are there any news on this?
